Makefile

```diff
@@ -225,6 +225,20 @@ clean:
 protobuf:
 	$(MAKE) -C .. -f cockroach/build/protobuf.mk
 
+# The .go-version-check target is phony so that it is rebuilt every
+# time, but we create a .go-version-check file so that the include
+# directive below only complains about .go-version-check not existing
+# the first time we are run.
+.PHONY: .go-version-check
+.go-version-check:
+	@required=go1.6; \
```

Comment:
Hmm, yeah, I can put the version in a separate file. That would also eliminate the warning when make is invoked and `.go-version-check` cannot be found. Not sure I want to rollback the changes to `sql/parser/Makefile`. Regenerating the files there is fast. There isn't much reason to try and optimize that.